nixos/modules/services/networking/tayga.nix

```diff
@@ -103,13 +105,20 @@ in
       dataDir = mkOption {
         type = types.path;
         default = "/var/lib/tayga";
-        description = "Directory for persistent data";
+        description = "Directory for persistent data.";
       };
 
       tunDevice = mkOption {
         type = types.str;
         default = "nat64";
-        description = "Name of the nat64 tun device";
+        description = "Name of the nat64 tun device.";
+      };
+
+      mappings = mkOption {
+        type = types.listOf types.str;
```

Comment:
Would it potentially make more sense for the type to be an `attrsOf str`? Or is it possible to have multiple IPv6 addresses mapped to the same IPv4 address?

Reply:
Neither IPv4 nor IPv6 addresses are accepted twice in the mapping by tayga. So I've changed the module and added a check.